src/components/tools/table-tool/table-tool.tsx

```diff
@@ -47,8 +48,8 @@ const TableToolComponent: React.FC<IToolTileProps> = observer(({
   } = useModelDataSet(model);
 
   // Set up user specified columns and function to measure a column
-  // TODO The user specified columns should be moved out of react and into MST
-  const { userColumnWidths, measureColumnWidth } = useMeasureColumnWidth();
+  const { measureColumnWidth, resizeColumn, resizeColumnWidth }
+    = useMeasureColumnWidth({ content });
```

Comment:
nit: seems like this should fit on one line now.
```suggestion
  const { measureColumnWidth, resizeColumn, resizeColumnWidth } = useMeasureColumnWidth({ content });
```